portfolio.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Portfolio</title>
+  <link href="./styles/normalize.css" rel="stylesheet">
+  <link href="./styles/icono.min.css" rel="stylesheet" >
+  <link href="./styles/main.css" rel="stylesheet">
+  <script src="https://use.fontawesome.com/ed5b29bc68.js"></script>
+</head>
+
+<body>
+  <header>
+    <li><a href="index.html"><i class="icono-home"></i></a></li>
+    <nav>
+      <li><a href="portfolio.html"> My Pieces </a></li>
+      <li><a href="code-journal.html"> Learning </a></li>
+      <li><a href="about.html"> Resume </a></li>
+    </nav>
+  </header>
+  <section id="portfolio">
+    <h1> Work Pieces </h1>
+    <article>
+      <h2><a href="https://github.com/Sofia15/ride-share-two">ride-share-two</a></h2>
+      <p>
+        Created a system that will manage information related to ride sharing from a CSV files. The system should have a way to keep track of drivers, riders and the trips that the drivers take with their riders by using Ruby.
+      </p>
+    </article>
+    <article>
+      <h2><a href="https://github.com/Sofia15/meowspace">meowspace</a></h2>
+      <p>
+        HTML&CSS practice project where I practiced with a given wireframe.
+      </p>
+    </article>
+    <article>
+      <h2><a href="https://github.com/Sofia15/Scrabble">Scrabble</a></h2>
+      <p>
+        Using Ruby, Agile work methodology and Test Driven Development (TDD), developed scrabble game as pair programming.
+      </p>
+    </article>
+    <article>
+      <h2><a href="https://github.com/Sofia15/Calculator">Calculator</a></h2>
+      <p>
+        Ruby - built a calculator command line interface (CLI) that allows a user to perform simple arithmetic.
+      </p>
+    </article>
+    <h3>To be endlessely contirnued ..</h3>
```

Comment:
There is no easy way to do this (maybe a linter of some sort?) but be mindful of spelling and grammar errors.